client/src/pages/HandleAuth.js

```diff
@@ -31,16 +32,16 @@ const HandleAuth = (props) => {
 
   let text;
   if (isMagicLinkValid == true) {
-    text = <p>Magic link is valid</p>;
+    return <Redirect to="/admin" />
   } else {
-    text = <p>Magic link is NOT valid</p>;
+    return (
+      <div className="flex-container">
+        <div>Sorry, magic link is not valid</div>
```

Comment:
If you are going for longer term, you might want to write "Sorry that link is not valid.". I am assuming that not every user will know what a magic link is. Either way, I am approving this PR. Feel free to merge as you see fit.

Reply:
Good catch, thanks!